package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "compile": "tsc index.ts test/spec.ts --module commonjs --noImplicitAny",
     "coverage": "cat ./coverage/lcov.info | coveralls",
-    "test": "istanbul cover _mocha",
+    "test": "istanbul cover node_modules/mocha/bin/_mocha",
```

Comment:
Hm, it seems that there is a better fix proposed in a bug report such as
```
$ istanbul cover --hook-run-in-context _mocha -- -R spec
```

I can replace it if you are okay with that.